web/packages/shared/components/MenuAction/MenuActionIcon.tsx

```diff
@@ -53,9 +53,10 @@ export default class MenuActionIcon extends React.Component<Props> {
           onClick={this.onOpen}
           data-testid="button"
         >
-          <Ellipsis size="medium" />
+          <MoreHoriz size="medium" />
         </ButtonIcon>
         <Menu
+          getContentAnchorEl={null}
```

Comment:
While working on Connect My Computer I noticed that `MenuActionIcon` is not positioned correctly. 
The menu was always placed in the center of anchor element, ignoring the vertical alignment. 
We already add `getContentAnchorEl={null}` to `Menu` component in most places, so it isn't a new fix.